src/managers/input/InputManager.cpp

```diff
@@ -421,7 +421,7 @@ void CInputManager::mouseMoveUnified(uint32_t time, bool refocus) {
             m_bLastFocusOnLS = false;
             return; // don't enter any new surfaces
         } else {
-            if (((FOLLOWMOUSE != 3 && allowKeyboardRefocus) && (*PMOUSEREFOCUS || m_pLastMouseFocus != pFoundWindow)) || refocus) {
+            if (allowKeyboardRefocus && ((FOLLOWMOUSE != 3 && (*PMOUSEREFOCUS || m_pLastMouseFocus != pFoundWindow)) || refocus)) {
```

Comment:
The idea is only allow 'refocus' if allowKeyboardRefocus is true.

(I could have changed `refocus` to `(allowKeyboardRefocus && refocus)`, but I just extracted allowKeyboardRefocus to the front since it's required in both cases)